test/test_jit.py

```diff
@@ -5128,7 +5128,11 @@ def test_math_ops(self):
         def test_floor():
             return math.floor(1.5)
 
+        def test_sinh():
```

Comment:
Do you mind refactoring it so both floor and sinh take in a parameter, and testing the result with a few different values ? With floor, you should add a negative input.